components/header.js

```diff
@@ -0,0 +1,38 @@
+var yo = require('yo-yo')
+
+module.exports = function (state, onaction) {
+  var views = {}
+
+  views.home = function () {
+    return yo`<nav class="bar bar-nav">
+        <a class="icon icon-plus pull-left" onclick=${ function () { onaction('add') } }></a>
+        <h1 class="title">Dat</h1>
+        <a class="icon icon-gear pull-right" onclick=${ function () { onaction('settings') } }></a>
+      </nav>`
+  }
+
+  views.add = function () {
+    return yo`<nav class="bar bar-nav">
+        <a class="icon icon-left-nav pull-left" onclick=${ function () { onaction('home') } }></a>
+        <h1 class="title">Add</h1>
+      </nav>`
+  }
+
+  views.settings = function () {
+    return yo`<nav class="bar bar-nav">
+        <a class="icon icon-left-nav pull-left" onclick=${ function () { onaction('home') } }></a>
+        <h1 class="title">Settings</h1>
```

Comment:
I wonder if we could have a 'default header' that took a parameter for the title, since that's the only thing that's changing in add/settings/detail?